src/shared/constants.cpp

```diff
@@ -13,13 +13,23 @@
 
 namespace {
 bool s_inProduction = true;
+QString s_versionOverride = "";
 }  // namespace
 
 bool Constants::inProduction() { return s_inProduction; }
 
 void Constants::setStaging() { s_inProduction = false; }
 
-QString Constants::versionString() { return QStringLiteral(APP_VERSION); }
+void Constants::setVersionOverride(const QString& versionOverride) {
```

Comment:
Revert this and see if we can use `InspectorHandler::appVersionForUpdate`

Reply:
That doesn't work, because it doesn't override the actual version constant which is what the addon API uses.

Reply:
can we make it happen? I'm OK with your approach if we remove the other one.

Reply:
Sure, the other one doesn't seem to be used anyways.

Reply:
Hm, so that is not used in tests but doesn't QA use it for something else? I can file a follow up to look into this, if you prefer. It is unrelated from this PR.

Reply:
They use it to test the update, forcing a lower version value.

Reply:
Alright so that mean we can't remove it.